src/domain/services/logger.service.ts

```
@@ -0,0 +1,27 @@
+  send: () => Promise<null>;
+
+  setLabels: (labels: Object) => void;
+}
+
+export namespace LoggerService {
+  export type Params = {
+    metadata?: Object;
+  };
+  export type Model = any;
+
+  export type Buffer = [string, metadata?: Object];
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)